scripts/windows/build/Publish-Branch.ps1

```diff
@@ -151,6 +151,14 @@ foreach ($appProjectFileName in $appProjectList) {
     if ($LASTEXITCODE -ne 0) {
         throw "Failed app publishing $appProjectFilePath."
     }
+    
+    # Copy rocksdb ARM32 version to native/arm folder; this rocksdb.dll statically linked with snappy dll.
+    if ($appProjectFileName -eq "Microsoft.Azure.Devices.Edge.Agent.Service.csproj" -or $appProjectFileName -eq "Microsoft.Azure.Devices.Edge.Hub.Service.csproj")
+    {
+        $rocksdbARMDestPath = Join-Path $ProjectPublishPath "native\arm"
+        $rocksdbARMSourcePath = Join-Path $BuildRepositoryLocalPath "thirdparty\rocksdb\arm32\rocksdb.dll"
+        Copy-Item $rocksdbARMSourcePath $rocksdbARMDestPath -Force
```

Comment:
Is this happening even in non-ARM builds?

Reply:
Yes.  Currently build will generate native with amd64 and i386 folder there; as runtime option (-r) is not provided when running dotnet publish; therefore it generated all different runtimes.